src/main/java/core/basesyntax/service/impl/CsvFileWriterService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvFileWriterService implements FileWriterService {
```

Comment:
```suggestion
public class FileWriterServiceImpl implements FileWriterService {
```
we can write to any file with this service, not only csv